README.md

```diff
@@ -2,16 +2,16 @@
 
 <strong>** DEV BRANCH **</strong>
 
-<strong>Images built from this branch are NOT guarenteed to always work.</strong>
+<strong>Images built from this branch are NOT guaranteed to always work.</strong>
 
-Run ovos on top of RaspberryPiOS
+Run OVOS on top of RaspberryPiOS
 
-Prebuilt images can be found on the [OpenVoiceOS Downloads Page](https://downloads.openvoiceos.com/images/raspbian/)
+Prebuilt images will be found on the [OpenVoiceOS Downloads Page](https://downloads.openvoiceos.com/images/raspbian/) once our image server is operational.
```

Comment:
I have a server setup for this now through the github actions.  The address will be 
https://ovosimages.ziggyai.online/raspbian

Reply:
references to ziggy ai added to README.md, manual_user_install.sh, and install_rasbpian.md (in documentation)